src/publish.js

```diff
@@ -32,7 +32,7 @@ export const queue = new PQueue({ concurrency: CONCURRENCY })
  * @param {string} b64record Base 64 encoded serialized IPNS record.
  * @returns {undefined}
  */
-export async function addToQueue (key, value, b64record) {
+export function addToQueue (key, value, b64record) {
```

Comment:
Don't we need addToQueue to be async since we are now await-ing publishRecord?

Reply:
`addToQueue` doesn't actually call `publishRecord`, it just defines the nested `run` function and passes it to `queue.add`. The calling of `publishRecord` happens inside `run`.

Reply:
Got it 👍